marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/system/finishHubDeployment.sjs

```diff
@@ -17,5 +17,5 @@
 
 // No privilege required: For now it is only used during installation and installation requires admin privileges.
 
-const jobQueryLib = require("/data-hub/5/jobs/job-query-lib.sjs");
+const jobQueryLib = require("/data-hub/5/flow/jobQueryLib.sjs");
```

Comment:
Let's stick with job-query-lib.sjs as the name - the naming isn't consistent in the DHF ML code, but things like entity-lib and hub-utils are widely used, so "worm case" is safe to use

Reply:
Though "jobQueryLib" is good as a prefix, which is consistent with entityLib as a prefix.